taskcluster/ci/ui-test/kind.yml

```
@@ -66,3 +66,17 @@ jobs:
                 - [automation/taskcluster/androidTest/ui-test.sh, x86-screenshots-tests, app.apk, android-test.apk, '-1']
         treeherder:
             symbol: debug(screenshots-x86)
+    x86-beta:
+        attributes:
+            build-type: beta-firebase
+        description: Test Fenix
+        run-on-tasks-for: [github-push]
+        run-on-git-branches: ["^releases/.+$", "^releases_.+$"]
```

I'm sorry, I didn't provide the best regex. This one is equivalent and looks simpler to me.

```suggestion
        run-on-git-branches: ["^releases[/_].+$"]
```